app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestPresenter.java

```diff
@@ -84,7 +85,13 @@ public void addCategory(Category category) {
 
     @Override
     public void onNewCategoryAdded(Intent data) {
-        //todo - Add category
+        String categoryName = data.getStringExtra("NAME");
+        String categoryDescription = data.getStringExtra("DESCRIPTION");
```

Comment:
Yeah I agree. I actually revisited this and changed it in a later branch, but forgot to go back here and update too.